validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/DiscoverySequences.java

```diff
@@ -331,12 +331,14 @@ private void scanAndVerify(Date scanStart, DiscoveryScanMode shouldEnumerate) {
 
     checkThat("discovery events were valid", reasons.isEmpty(), CSV_JOINER.join(reasons));
 
+    /* Allow duplicate scan addresses
```

Comment:
Erm? Why this?  Duplicate addresses should be filtered out client side...

Reply:
For my case multiple devices running on a single host, requiring them to share the same IPv4 and BACnet addresses. They indeed different virtual gateways sharing same network from underlying host.